src/test/scala/edu/ie3/simona/model/participant/PvModelSpec.scala

```diff
@@ -0,0 +1,802 @@
+        (
+          35.0d,
+          +23.2320597d,
+          +74.648850625d,
+          45d,
+          -90d,
+          109.00780288303966d,
+        ), // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to EAST at 17:00
+        (
+          27.96d,
+          -17.51d,
+          -11.1d,
+          30d,
+          +10d,
+          22.384603601536398d,
+        ), // Goswami Priciples of Solar Engineering Example 2.7a
+        (
+          -35.3d,
+          -17.51d,
+          -4.2d,
+          30d,
+          +170d,
+          14.882390116876563d,
+        ), // Goswami Priciples of Solar Engineering Example 2.7b
+      )
+
+      // Calculate the angle of incidence of beam radiation on a surface located at a Latitude
+      // at a certain hour angle (solar time) on a given declination (date) if the surface
+      // is tilted by a certain slope from the horizontal and pointed to a certain panel azimuth
+      // west of south.
+      forAll(testCases) {
+        (latitudeDeg, deltaDeg, omegaDeg, gammaEDeg, alphaEDeg, thetaGOut) =>
+          // When
+          val thetaG = pvModel.calcAngleOfIncidenceThetaG(
+            Degrees(deltaDeg),
+            Degrees(latitudeDeg),
+            Degrees(gammaEDeg),
+            Degrees(alphaEDeg),
+            Degrees(omegaDeg),
+          )
+
+          // Then
+          thetaG.toDegrees shouldEqual thetaGOut +- 1e-10
+      }
+    }
+
+    "deliver equal angles of incidence for special case" in {
+      val testCases = Table(
+        (
+          "latitudeDeg",
+          "deltaDeg",
+          "omegaDeg",
+          "gammaEDeg",
+          "alphaEDeg",
+          "thetaGOut",
+        ),
+        (45d, -7.15, -82.5d, 60d, 0d, 80.94904834048776d), // thetaG
+        (45d, -7.15d, -82.5d, 0d, 0d,
+          89.79565474295107d), // Latitude 45 degrees, slope 0 degrees (zenith angle)
+        (15d, -7.15, -82.5d, 30d, 0d,
+          80.94904834048776d), // same test but 30° South with 30° less sloped surface
+        (0d, -7.15, -82.5d, 15d, 0d,
+          80.94904834048776d), // same test but 15° South with 15° less sloped surface
+        (-15d, -7.15, -82.5d, 0d, 0d,
+          80.94904834048776d), // same test but 15° South with 15° less sloped surface
+        (-30d, -7.15, -82.5d, 15d, 180d,
+          80.94904834048776d), // same test but 15° South with 15° more sloped surface (Surface is now facing north, since it is in the southern hemisphere, therefore the surface azimuth is 180°)
+        (52.3d, 23.4337425, 2.15114395d, 0d, 0d,
+          28.91315041538251d), // Berlin 21.06. 12:00 => thetaG = 90 - alphaS
+        (70.3d, 23.4337425, 2.15114395d, 18d, 0d,
+          28.91315041538251d), // same test but 18° North with 18° sloped surface(40d, -11.6d, 82.5d, 60d, 0d, 79.11011928744357d),
+        (40d, -11.6d, 82.5d, 60d, 0d, 79.11011928744357d),
+        (40d, -11.6d, -78.0d, 60d, 0d, 74.92072065185143d),
```

Comment:
Thanks for merging the tests. I'd still leave in the tests with "inverted omegas" that you left out for clarity though...

Reply:
I readded the inverted omegas. I also moved them to the function "calculate angle of incidence", since they shouldn't be in the ´deliver correct angles of incidence at different latitudes and slope angles´ function in my opinion